src/app/layout.tsx

```diff
@@ -9,14 +9,31 @@ import { getServerSession } from "next-auth";
 import { getLocale } from "./functions/server/l10n";
 import { SessionProvider } from "../contextProviders/session";
 import { authOptions } from "./api/auth/[...nextauth]/route";
+import { getL10n } from "./functions/server/l10n";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export const metadata = {
-  title: "Firefox Monitor",
-  description:
-    "Find out if you’ve been part of a data breach with Firefox Monitor. We’ll help you understand what to do next and continuously monitor for any new breaches.",
-};
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: l10n.getString("brand-fx-monitor"),
+    description: l10n.getString("meta-desc-2"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
```

Comment:
I can confirm these w/ the Twitter+OpenGraph linters once we get this branch on a dev/staging server, but I'm pretty sure these images need to be fully qualified (if they don't magically get converted somewhere in the pipeline already).
Here's the old version (L31 and L37 below):
https://github.com/mozilla/blurts-server/blob/d4cb77862d91a61e0debf1032301428c2019ee04/src/views/guestLayout.js#L31-L37

Reply:
It does indeed seem to be magically fully qualified, I get `http://localhost:6060/...` for the `image` properties.

Reply:
https://nextjs.org/docs/app/api-reference/file-conventions/metadata/opengraph-image also has a neat bit about custom opengraph and twitter images (although no mentions of them supporting .webp).

> Add an `opengraph-image.(jpg|jpeg|png|gif)` image file to any route segment.
> Add a `twitter-image.(jpg|jpeg|png|gif)` image file to any route segment.

And it will add a bunch of the meta tags automatically:
```html
<meta property="og:image" content="<generated>" />
<meta property="og:image:type" content="<generated>" />
<meta property="og:image:width" content="<generated>" />
<meta property="og:image:height" content="<generated>" />

<meta name="twitter:image" content="<generated>" />
<meta name="twitter:image:type" content="<generated>" />
<meta name="twitter:image:width" content="<generated>" />
<meta name="twitter:image:height" content="<generated>" />
```

Can't see mentions of webp specifically on [Open Graph protocol](https://ogp.me/) specs, and this looks possibly too outdated to be definitive: https://stackoverflow.com/questions/57955612/can-we-use-webp-extension-image-in-ogimage-meta-tag.

---

In fact, I see we're using .webp already in production, so if I go to the Meta Sharing Debugger at https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fmonitor.firefox.com it seems to throw a vague warning:

> ### ⚠️ Warnings That Should Be Fixed
> **Image Unavailable**
> Provided `og:image`, https://monitor.firefox.com could not be downloaded. This can happen due to several different reasons such as your server using unsupported content-encoding. The crawler accepts deflate and gzip content encodings.